recipes/mpir/all/conanfile.py

```diff
@@ -62,6 +62,8 @@ def _configure_autotools(self):
                 args.extend(['--disable-static', '--enable-shared'])
             else:
                 args.extend(['--disable-shared', '--enable-static'])
+            if self.options.fPIC:
+                args.extend(['--with-pic'])
```

Comment:
Doesn't this suffice? https://github.com/conan-io/conan/blob/ed6b33681d1cf25e486518400d0bd845bf5c9aa1/conans/client/build/autotools_environment.py#L326-L327

Reply:
```suggestion
            args.append(["--with-pic" if self.options.get_safe("fPIC") else "--without-pic"])
```
This should fix a potential problem for mingw.

Reply:
Using the code above (without the brackets) works.
`args.append("--with-pic" if self.options.get_safe("fPIC") else "--without-pic")`

https://github.com/conan-io/conan/blob/ed6b33681d1cf25e486518400d0bd845bf5c9aa1/conans/client/build/autotools_environment.py#L326-L327
does not seem to be sufficient in my case.